### DISCUSSION TITLE
Merge Lines

### DISCUSSION THREAD

**Discussion by v0idp** [ocornut/imgui#4087] [Old/Archived discussions]
is there a possibility to merge lines into one big line with n amount of points or is there any other function which is more useful for that ?

**Comment by rokups** [comment]
Sounds like you are looking for `ImDrawList::AddPolyline()`.

**Reply by v0idp** [reply]
looking at the underlying code, addline uses addpolyline right ? So performance-wise it's the same ?

**Reply by rokups** [reply]
Yes
